modules/core/src/test/java/org/apache/ignite/testframework/junits/common/KeyType.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.ignite.testframework.junits.common;
+
+/**
+ * Defined type of key
+ * */
+public enum KeyType {
+    PRIMARY(0),
```

Comment:
Do we really need `getKeyIndex`?
We can use `Enum#ordinal` instead of `getKeyIndex`.

Reply:
It’s not a good practice to be tied to the order of enums.

Reply:
We should provide solutions as small/simple as possible.
In case you're not required this now, code should not contain this.
For every code fragment at PR, you should be able to provide proper explanation"why it's here", "good practice" is not acceptable explanation.

BTW, `keyType.getKeyIndex() * 100_00` looks odd to me, can we refactor this to?